modules/va_facilities/app/controllers/va_facilities/v0/facilities_controller.rb

```diff
@@ -100,15 +100,6 @@ def verify_float(param)
         raise Common::Exceptions::InvalidFieldValue.new(param.to_s, params[param])
       end
 
-      def validate_bbox
```

Comment:
If this method is being removed, why is it still being called in the `validate_params` method?

Reply:
For that matter, why is this method being removed?